<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>被识别概率计算软件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bu_exit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bu_deduction">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>近远场推演模块</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bu_detection">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>发射装备检测识别模块</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bu_probability">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>目标识别概率计算模块</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
